packages/docs/source/components/status.md

```diff
@@ -0,0 +1,167 @@
+# Status
+
+Status is used to keep users informed, by providing appropriate feedback on particular concepts or entities. For Okta, status can be used to display overall operational status all the way down to very granular concepts, like user status.
+
+## Usage
+
+Status should be used to communicate the state of a discrete item, such as a server or individual process. Both labeled and unlabeled variants are acceptable, but should follow the associated guidelines below.
+
+<figure class="nimatron--example">
+  <div class="nimatron--rendered">
+    <span class="ods-status--label">Server status</span>
+    <span class="ods-status is-ods-status-success">
+      <span class="u-visually-hidden">Server status</span>
+      System operational
+    </span>
+  </div>
+
+  ```html
+  <span class="ods-status--label">Server status</span>
+  <span class="ods-status is-ods-status-success">
+    <span class="u-visually-hidden">Server status</span>
+    System operational
+  </span>
+  ```
+</figure>
+
+If a label is not present, ensure that appropriate context is communicated by the Status itself and close proximity to the content that it’s supporting.
+
+In either case, be sure to include screen-reader only text to ensure the appropriate context is available for users who rely on assistive technology.
+
+<figure class="nimatron--example">
+  <div class="nimatron--rendered">
+    <span class="ods-status is-ods-status-success">
+      <span class="u-visually-hidden">Server status</span>
+      System operational
+    </span>
+  </div>
+
+  ```html
+  <span class="ods-status is-ods-status-success">
+    <span class="u-visually-hidden">Server status</span>
+    System operational
+  </span>
+  ```
+</figure>
+
+## Variants
+
+There are four variants of Status in Odyssey: Neutral, Success, Caution, and Danger.
```

Comment:
Do we plan on extending these `Status` types to map to other components? I see that `success` and `danger` types are reused, but `caution` and `neutral` seem newer.

Reply:
Yup, we'll be using the color semantics as the baseline for variant naming across Odyssey - Neutral and Caution will trend more rare than Action, Danger, and Success.

Reply:
@jmelberg-okta Status' will also be a new content type in tables.